### ISSUE TITLE
feature importance: option to stack weights for categorical variable values 

### ISSUE THREAD

**Issue by arsenyinfo** [TeamHG-Memex/eli5#159]
We discussed with @kmike and @lopuhin that it may be useful for analyzing datasets with categorical variables. Most likely it should be an additional wrapper function or an option for existing functions - TBD.

I'm going to work on this issue in a week.

**Comment by kmike** [comment]
If we're talking about the user-facing API, I think an option for existing functions would be better - it'd be more consistent with the rest of eil5.
